Postman/Surveys.postman_collection.json

```diff
@@ -0,0 +1,353 @@
+{
+	"info": {
+		"name": "Surveys",
+		"_postman_id": "5f16b4a3-0f45-6164-b249-9a9993aea86b",
+		"description": "A controller for querying Surveys",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Sign In",
```

Comment:
could we move sign in to a separate collection instead of repeating it?

Reply:
Yes, we can! :-)